cmd/rpcdaemon/cli/config.go

```diff
@@ -564,7 +564,7 @@ func isWebsocket(r *http.Request) bool {
 // a new secret and stores to the default location.
 func obtainJWTSecret(cfg httpcfg.HttpCfg) ([]byte, error) {
 	var fileName string
-	if len(cfg.JWTSecretPath) > 0 {
+	if cfg.JWTSecretPath != JwtDefaultFile {
```

Comment:
This `if` is pointless now. If you just keep the line 572 it will be the same logic with less code.

Reply:
I got ride of the filename var since it is useless now too